bare-metal/elastic-metal/troubleshooting/enable-ssh-virtualization-distribution.mdx

```diff
@@ -54,6 +54,10 @@ If you require SSH access to your Elastic Metal server, you can enable SSH on yo
 
 ## Enabling SSH on ESXi
 
+<Message type="important">
+  VMware has been acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn more [here](/tutorials/configure-failover-proxmox/).
```

Comment:
```suggestion
  VMware was acquired by Broadcom and free ESXi licenses have been discontinued. For details, see [here](https://knowledge.broadcom.com/external/article?legacyId=2107518). This content is provided as-is with no guarantee of functionality. Consider using alternative hypervisors like Proxmox. Learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).
```

Reply:
Due to VMware's acquisition by Broadcom, the free edition of VMware vSphere Hypervisor is no longer available on the VMware website. For more details, check out [the EOL of free vSphere Hypervisor](https://knowledge.broadcom.com/external/article?legacyId=2107518). Please note that the following content is provided as-is, without any guarantee of functionality.
We suggest considering alternative hypervisors such as Proxmox. You can learn how to [deploy virtual machines with Proxmox on Scaleway Dedibox](/tutorials/configure-failover-proxmox/).